src/extension.ts

```diff
@@ -163,6 +163,33 @@ const createModules = (document: vscode.TextDocument, position: vscode.Position)
   return [];
 };
 
+const createHover = (document: vscode.TextDocument, position: vscode.Position): vscode.Hover => {
+  const text = document.getText(document.getWordRangeAtPosition(position));
+  const rm = new RegExp('([a-zA-Z0-9$_ ]+)\\.' + text + '([a-zA-Z0-9$_ ]*)');
+  const match = rm.exec(document.lineAt(position.line).text);
+  const modules = Object.keys(iotjs);
+  let hoverContent = Array<vscode.MarkdownString>();
+  let hoverText: string;
+  let hoverModule: string;
+  const availableModules = defaultModules.concat(lookForModules(document.getText()));
+
+  availableModules.forEach(mod => {
+    if ( match[1] === mod.link) {
+      hoverText = text;
+      hoverModule = mod.mod;
+    }
+  });
```

Comment:
I would use a similar pattern like this over here:
```typescript
const { hoverText, hoverModule } = availableModules.find(mod => mod.link === match[1]).map(mod => ({
  hoverText: text,
  hoverModule: mod.mod
});
```
in tis case we can eliminate the `let`.